pkg/retriever/graphsyncretriever.go

```diff
@@ -323,21 +304,16 @@ func runRetrievalCandidate(
 				)
 			}
 		} // else we didn't get to retrieval phase because we were cancelled
-	} // else we didn't get to retrieval phase because query failed
-
+	} // else we did't get to the retrieval phase cause query failed
```

Comment:
```suggestion
	}

```

(this is the else now)